cartridge/roles/metrics.lua

```diff
@@ -46,11 +46,16 @@ local function set_labels(custom_labels)
     metrics_vars.custom_labels = custom_labels
 end
 
-local function check_config(_)
+local function check_config(config)
     checks({
         export = 'table',
         ['global-labels'] = '?table',
+        include = '?table',
+        exclude = '?table',
     })
+    if config.include and config.exclude then
+        error("don't use exclude and include sections together", 0)
```

Comment:
Can't understand 2 things:
1. Why we need both include and exclude? One of the ways is to use only one single list
2. If use include and exclude  why not togather?

Reply:
1) Yes, it's one possible way, but my point is to suggest users to choose: include only metrics they want, or include all metrics from default metrics list except metrics they don't need
2) That sections has separate logic. If you use both sections, which section should has higher priority?